### PR TITLE
prest: New port

### PR DESCRIPTION
**Pull request by judaew**
Version: 1.0.8

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.2.3 20D91
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by judaew** [comment]
Can you help to explain what the error is that causes the checks to fail?

File: devel/prest/Portfile [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    pREST (PostgreSQL REST), simplify and accelerate development, instant, \
+    realtime, high-performance on any Postgres application, existing or new
+homepage            https://prestd.com
+
+checksums           ${distname}${extract.suffix} \
+                        rmd160  448a195892366ae2fd5a07888a9c3bb285ffee4f \
+                        sha256  0751ceb27bf9e32cbc67a6e01f657b9532b9eac6aa6198f1929da2c56a3316c1 \
+                        size    97783
+
+build.args          "-o prestd cmd/prestd/main.go"
```
Comment (herbygillot) [inline devel/prest/Portfile:22]: You can drop the double quotes, and you don't have to specify the specific Go file as the build target.

The following should work:
```
build.args            -o prestd ./cmd/prestd
```

**Comment by herbygillot** [comment]
> Can you help to explain what the error is that causes the checks to fail?

https://dev.azure.com/macports/macports-ports/_build/results?buildId=13619&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=08703039-49d0-5b78-7691-5b9414c62c1d&l=1397

Looks like a vendored Go dependency whose Github repo URL is different than its module name.  This is a problem one runs into quite frequently when using `go2port`.

The error states that it can't find `jtolds-gls`, likely caused by this vendored dependency:  **github.com/jtolds/gls**

If you visit that URL, you will actually see the URL resolve to: _github.com/**jtolio**/gls_  (_jtolio_ instead of _jtolds_)

So you need to correct the name of the vendored dependency (to **github.com/jtolio/gls**), and you may have to adjust the checksum after doing so.  You may encounter this for several other vendored dependencies.

File: devel/prest/Portfile (hunk 1 above)
Comment (judaew) [reply devel/prest/Portfile]: Thanks 🌟

**Comment by judaew** [comment]
Oh, it's sly! I tried to fix the checksums before, but I didn't know to need another name of the dependency. Thanks so much.

**Comment by herbygillot** [comment]
Np.  Let's also ensure a `revision` field.

**Comment by judaew** [comment]
Done

**Comment by herbygillot** [comment]
Thanks @judaew

**Pull request merged**